services/Makefile

```diff
@@ -1,8 +1,8 @@
 deploy:
-	cd draw; fn deploy myapp
```

Comment:
I use this makefile quite often to push into remote registry... can you remove the --local? or create deploy-local?

Reply:
Ok, if you pass do `./setup.sh local`, it will do local only.